client/src/store/actions.ts

```diff
@@ -16,6 +18,16 @@ export const actions: ActionTree<State, State> = {
       console.log(error);
     }
   },
+  async [ActionTypes.AddRobotToServer]({ commit, state }, robot: Cart) {
```

Comment:
@taherkheli why call it `robot` if it is a `Cart`?

Reply:
done

Reply:
@Jaxwood  its actually a `robot` with an extra field `cost` appended to it. `Cart` is not a `cart `but an enhanced `robot `with price. Cart extends Robot Interface. Chief Architect JLo named it so, i did not dare changing it ... let it be I have had enough of robots :)